### ISSUE TITLE
Repeatable crash on turn 100 or above 

### ISSUE THREAD

**Issue by Cdarfoor** [LoneGazebo/Community-Patch-DLL#3474]
_1. Mod version (i.e Date - 4/23):_
08/23

_2. Mod list (if using Vox Populi only, leave blank):_
Vox Populi
(BNW) Leugi's Barbarian Inmersion Enhancements (v 1)
Barbarians Evolved (v 5)
Caravans Build Roads (v 1)
CBP-FW compatibility (v 1)
City States Evolved (v 3)
Civ Names by Policies (v 8)
Civ V Focused AI (v 1)
Cultural Capitals (Brave New World Edition) (v 16_5)
Cultural Diffusion (v 18)
Custom Advanced Setup Screen (v 5)
Economics (v 4)
Emigration (VP)
Faster Aircraft Animations (v 3)
Future Tech Does Something Beyond Score Increasing Mod (v 1)
Future Worlds (v 6)
Global Relations (v 7)
Global Warming (v 7)
Global Warming Congress (v 2)
Helicopters Can Fly Over Water (v 3)
JFD's Cities in Development (v 58)
JFD's Cultural Diversity (Core) (v 13)
JFD's Exploration Continued Expanded (v 20)
JFD's Rise to Power (v 33)
Map Labels (v 6)
Militia Defense (v 14)
Modern Realism (v 9)
More Active Religions (v 2)
Move Capital (v 10)
No Unit Limit (v 1)
Ocean Cities (v 2)
RazeAllCapitals (v 5)
Reforestation (v 9)
Resolution - International Monetary Fund (v 1)
Revolutions (v 9)
Slavery Mod (v 3)
Strange Religions (v 9)
Supercarrier (v 2)
UI for Unit Stacking (v 2)
Useful Oil Mod (v 10)
Water Tiles Enhanced (v 4)

_3. Error description:_ 
Amongst multiple save games I get always get a crash around turn 100 or beyond, I haven't been able to narrow down the root cause.

_4. Steps to reproduce (optional):_
Play up to or beyond turn 100-150+ and you should encounter it.
---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):
[Crash Logs.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/1254631/Crash.Logs.zip)

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):
[Test 1-c.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/1254632/Test.1-c.zip)

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steam
apps/common/Sid Meier's Civilization V)
[CvMiniDump.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/1254634/CvMiniDump.zip)
:

4. Screenshots (optional):

**Comment by TechpriestEnginseer** [comment]
There's just too much to diagnose sorry. :(

**Comment by ilteroi** [comment]
please try again with only half the 3rd party mods (and then the other half) to narrow down the culprit

**Comment by ScionOfEris** [comment]
He's not alone.  I've seen this mentioned on reddit/r/civvoxpopli, and have started hitting it myself.

I've just got 'vanilla' CBP installed,
[Crashing Civ Game.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/1264826/Crashing.Civ.Game.zip)

 so it is probably better for your purposes.

Mods:
community patch: v88
community balance overhaul v13
CS Diplomacy Mod for CBP v27
C4DF - CBP v11
More Luxuries CBO Edition v155
Community balance overhaul - Compat FIles (EUI) v1
Antiquities Site Icons Filter v1

I've got a save that crashes at every 'Next Turn'.  I turned on logging and made it crash, so I had logs/coredump to attach.  Attached are the logs, coredump and save.
